### ISSUE TITLE
A11y: Elements with a role attribute must have required parent element for the role

### ISSUE THREAD

**Issue by xiaolixu** [carbon-design-system/carbon#2612]
## Detailed description

> Describe in detail the issue you're having.
An element with WAI-ARIA role menuitem is not contained in or owned by an element with one of the following WAI-ARIA roles: group, menu, menubar. See rule help for remediation guidance.

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: IBM 4.1.2 Name, Role, Value
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/name_role_value.html
Test Tool Type: DAP

===== DEFECT DESCRIPTION =====
 An element with WAI-ARIA role identifier is not contained in or owned by an element with one of the following WAI-ARIA roles: identifier. See rule help for remediation guidance. 
    > An element with WAI-ARIA role menuitem is not contained in or owned by an element with one of the following WAI-ARIA roles: group, menu, menubar. See rule help for remediation guidance.

===== EXPECTED RESULT =====
For more detail:
https://aat.w3ibm.mybluemix.net/token/c3b7a795-a87f-4fe4-8071-059d47e0571f/b5cf2a6d-027b-42c7-aaa0-1e99adf7ac78/latest///doc/w3/help/en-US/idhi_accessibility_check_g1153.html

> Is this a feature request (new component, new icon), a bug, or a general issue?

a11y bug
> Is this issue related to a specific component?
OverflowMenu OverflowMenuItem
> What did you expect to happen? What happened instead? What would you like to see changed?

Elements with a role attribute must have required parent element for the role
> What browser are you working in?

chrome 68.0.3440.106 (Official Build) (64-bit)
> What version of the Carbon Design System are you using?

`    "carbon-components": "^9.5.0",
    "carbon-components-react": "6.21.4",`
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

virtual private cloud
## Steps to reproduce the issue
use OverflowMenu with _flipped floatingMenu_

**Comment by tw15egan** [comment]
No longer seeing this DAP error when testing in Storybook, as the roles have been added

<img width="385" alt="Screen Shot 2019-07-26 at 1 12 41 PM" src="https://user-images.githubusercontent.com/11928039/61978882-2e23ad00-afa7-11e9-93c4-669630fcba12.png">
